src/content/docs/guides/node-setup/deploy-a-node/index.mdx

```diff
@@ -125,15 +125,27 @@ rpm -i tenzir-static-amd64-linux.rpm
 </TabItem>
 <TabItem value="nix" label="Nix" icon="nix">
 
-Use our `flake.nix`:
+Use our `flake.nix` to run an ad-hoc tenzir process:
```

Comment:
```suggestion
Use our `flake.nix` to run an ad-hoc `tenzir` process:
```

Reply:
What does it even mean to the user to "run a Tenzir process"? A node? A pipeline? The example doesn't make sense to me.